Cargo.toml

```diff
@@ -5,7 +5,7 @@ homepage = "https://mintlayer.org"
 repository = "https://github.com/mintlayer/mintlayer-core"
 readme = "README.md"
 license = "MIT"
-version = "0.1.2"
+version = "0.2.1"
```

Comment:
Please revert this and the other one. We take care of bumping versions elsewhere.